src/Map.js

```diff
@@ -131,7 +131,7 @@ export default class Map {
     const mapOptions = {
       minZoom: this.minZoom,
       center: this.initialCenter,
-      zoomControl: false,
+      zoomControl: this?.zoomControl || false,
```

Comment:
I attempted to place the controls in the bottom left, but
 
      // NOTE: google maps script timing is causing window.google.maps.ControlPosition.BOTTOM_RIGHT to be undefined
      // https://developers.google.com/maps/documentation/javascript/examples/control-positioning
      // zoomControlOptions: {
      //   position: window.google.maps.ControlPosition.BOTTOM_RIGHT,
      // }